tests/cases/fourslash/completionListForStringUnion.ts

```diff
@@ -1,12 +1,11 @@
 /// <reference path='fourslash.ts' />
 
-//// type A = 'fooooo' | 'barrrrr';
+//// type A = 'foo' | 'bar' | 'baz';
 //// type B<T extends A> = {};
-//// type C = B<'fooooo' | '/**/'>
+//// type C = B<'foo' | '/**/'>
```

Comment:
Is this the only place the change is observable?

Reply:
This only affects completions in types, not values. I think this is the only place we test a string literal type completion.